src/verilog/expr2verilog.cpp

```diff
@@ -1956,6 +1987,9 @@ expr2verilogt::resultt expr2verilogt::convert_rec(const exprt &src)
   else if(src.id() == ID_verilog_value_range)
     return convert_value_range(to_verilog_value_range_expr(src));
 
+  else if(src.id() == ID_postincrement)
+    return convert_sva_unary(to_unary_expr(src), "++");
```

Comment:
Not a big deal, but am I correct that this is unrelated?

Reply:
It's needed to make the test work.